lib/user-interface/react/src/components/chatbot/Chat.tsx

```diff
@@ -295,6 +295,16 @@ export default function Chat({ sessionId }) {
       stop: modelConfig?.stop,
       modelKwargs: modelConfig?.modelKwargs,
     });
+    // LangChain's JS OpenAI client sets default values that we do not want if we didn't set them.
+    // The following parameters are preferred to be null instead of having a numeric default value
+    // as they degrade the quality of the model response if set incorrectly by default.
+    if (!modelConfig?.frequency_penalty && modelConfig?.frequency_penalty !== 0) {
```

Comment:
We should be able to add these inline above like this:

`
new ChatOpenAI({
      modelName: selectedModel?.id,
      openAIApiKey: auth.user?.id_token,
      configuration: {
        baseURL: `${RESTAPI_URI}/${RESTAPI_VERSION}/serve`,
      },
      streaming,
      maxTokens: modelConfig?.max_tokens,
      n: modelConfig?.n,
      topP: modelConfig?.top_p,
      frequencyPenalty: modelConfig?.frequency_penalty ?? null,
      presencePenalty: modelConfig?.presence_penalty ?? null,
      temperature: modelConfig?.temperature,
      stop: modelConfig?.stop,
      modelKwargs: modelConfig?.modelKwargs,
    });
`